packages/itwinui-react/src/core/InputGrid/InputGrid.tsx

```diff
@@ -21,19 +33,28 @@ type InputGridOwnProps = {
 //-------------------------------------------------------------------------------
 
 /**
- * InputGrid component is used to display inputs (input, textarea, select)
+ * InputGrid component is used to display form fields (input, textarea, select)
  * with label and/or status message
  *
- * @usage
+ * Form fields are automatically associated with the label and status message for
```

Comment:
Should this mention that if the user passes manual ids, that won't be overriden? Or maybe that can be implied?

Reply:
It's mentioned in the actual documentation